src/handlePR.js

```diff
@@ -25,15 +25,15 @@ const handlePR = async (
     repo,
     pull_number: prNumber,
     headers: {
-      accept: "application/vnd.github.v3.diff",
+      accept: "application/vnd.github.diff",
     },
   });
 
-  const changedLines = getChangedLines(isIgnored, pullRequestDiff.data);
+  const score = scoreChanges(isIgnored, pullRequestDiff.data);
```

Comment:
Perhaps instead of completely replacing the old implementation we should introduce configuration for scoring strategy? That way we can more easily experiment with even more strategies, or revert to an old strategy without re-releasing this action.

Reply:
I like that idea. Added here https://github.com/glortho/pr-size-helper-action/pull/3/commits/0fb440f914df2f995cd6dc471cbf6221a84ba5bc